### PR TITLE
Monitoring and alerting for Azimuth instances

### PR DESCRIPTION
**Pull request by mkjpryor**
Extends #50

### CODE REVIEW

File: roles/consul/tasks/main.yml [hunk 1]
```diff
@@ -52,55 +52,43 @@
...
+    consul_dashboard_definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: consul-grafana-dashboard
+        namespace: "{{ consul_release_namespace }}"
+        labels:
+          grafana_dashboard: "1"
+      data:
+        consul_dashboard.json: >-
```
Comment (sd109) [inline roles/consul/tasks/main.yml:117]: Have you tested that this dashboard renders properly in Grafana with the `>-` operator included? I seem to remember having some issues with this before.

File: roles/kube_prometheus_stack/defaults/main.yml [hunk 2]
```diff
@@ -1,172 +1,42 @@
...
+kube_prometheus_stack_release_namespace: monitoring-system
+kube_prometheus_stack_release_name: kube-prometheus-stack
+
+# The timeout to wait for the release to become ready
+kube_prometheus_stack_wait_timeout: 10m
+
+# The values for the kube-prometheus-stack release
+kube_prometheus_stack_release_defaults:
+  prometheus:
+    prometheusSpec:
```
Comment (sd109) [inline roles/kube_prometheus_stack/defaults/main.yml:18]: Do we want a persistent volume by default on single node deployments?

**Review by sd109 — APPROVED** [review]

File: roles/consul/tasks/main.yml (hunk 1 above)
Comment (mkjpryor) [reply roles/consul/tasks/main.yml:117]: It seems to

File: roles/kube_prometheus_stack/defaults/main.yml (hunk 2 above)
Comment (mkjpryor) [reply roles/kube_prometheus_stack/defaults/main.yml:18]: My feeling is that single nodes are mostly for development. Having a persistent volume seems like overkill.

Comment (mkjpryor) [reply roles/kube_prometheus_stack/defaults/main.yml:18]: We can always add one down the line if it turns out to be an issue.

**Pull request merged**